central/complianceoperator/v2/report/manager/manager_impl.go

```diff
@@ -103,10 +104,11 @@ func (m *managerImpl) SubmitReportRequest(ctx context.Context, scanConfig *stora
 	}
 
 	req := &reportRequest{
-		scanConfig: scanConfig,
-		ctx:        context.WithoutCancel(ctx),
+		scanConfig:         scanConfig,
+		ctx:                context.WithoutCancel(ctx),
+		notificationMethod: method,
 	}
-	log.Infof("Submitting report for scan config %s at %v for execution with req %v.", scanConfig.GetScanConfigName(), time.Now().Format(time.RFC822), *req)
+	log.Infof("Submitting report for scan config %s at %v with method %s for execution with req %v.", scanConfig.GetScanConfigName(), time.Now().Format(time.RFC822), method.String(), *req)
```

Comment:
```suggestion
	log.Infof("Submitting report for scan config %s at %s with method %s for execution with req %v.", scanConfig.GetScanConfigName(), time.Now().Format(time.RFC822), method.String(), *req)
```

time.Format returns string

Reply:
The `%v` and `*req` may be pretty chatty (and id duplicates the method) - consider printing less, or moving to debug.

Reply:
I changed it to not print the request.